src/solc_0.8/assetERC1155/AssetERC1155.sol

```diff
@@ -2,19 +2,26 @@
 pragma solidity 0.8.2;
 
 import "./AssetBaseERC1155.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {
+    DefaultOperatorFiltererUpgradeable
+} from "../OperatorFilterer/contracts/upgradeable/DefaultOperatorFiltererUpgradeable.sol";
+import {OperatorFiltererUpgradeable} from "../OperatorFilterer/contracts/upgradeable/OperatorFiltererUpgradeable.sol";
 
 // solhint-disable-next-line no-empty-blocks
-contract AssetERC1155 is AssetBaseERC1155 {
+contract AssetERC1155 is AssetBaseERC1155, OperatorFiltererUpgradeable {
```

Comment:
if our goal is to upgrade mainnet asset, then we shouldn't use this contract but the 0.5 one